test/e2e/snaps/ppom-blockaid-alert.spec.js

```diff
@@ -0,0 +1,192 @@
+            ],
+          },
+        ];
+
+        for (const config of testBenignConfigs) {
+          const { logExpectedDetail, method, params } = config;
+
+          // Send JSON-RPC request
+          const request = JSON.stringify({
+            jsonrpc: '2.0',
+            method,
+            params,
+          });
+          await driver.executeScript(
+            `window.transactionHash = window.ethereum.request(${request})`,
+          );
+
+          // Wait for confirmation pop-up
```

Comment:
To fix the first test we should add here this line, otherwise Chrome goes faster and fails:

`await driver.waitUntilXWindowHandles(3);`

![Screenshot from 2023-09-29 10-47-22](https://github.com/MetaMask/metamask-extension/assets/54408225/08d08e4b-4138-4a56-95c7-ab045cfcec79)